include/dem/dem.h

```diff
@@ -234,8 +235,9 @@ class DEMSolver
    * @param dem_parameters DEM parameters
    * @return A pointer to the particle-wall contact force object
    */
-  std::shared_ptr<PWContactForce<dim>>
-  set_pw_contact_force(const DEMSolverParameters<dim> &dem_parameters);
+  std::shared_ptr<ParticleWallContactForce<dim>>
+  set_particle_wall_contact_force(
+    const DEMSolverParameters<dim> &dem_parameters);
```

Comment:
Could this function be moved outside of the DEM solver, this way it would be shared with the CFD-DEM code?

Reply:
I'll do this in another PR.